.vib/keycloak/cypress/cypress/integration/keycloak_spec.js

```diff
@@ -4,30 +4,36 @@ import { random } from '../support/utils';
 it('allows creating a new user', () => {
   cy.login();
   cy.get('[href*="/users"]').click();
-  cy.get('#createUser').click();
+  cy.get('button[data-testid="add-user"]').click();
```

Comment:
You can probably simplify this a bit:

```suggestion
  cy.get('[data-testid="add-user"]').click();
```

Same with the rest of similar selectors.

Reply:
I didn't know this. Thanks!